pulpcore/cli/core/artifact.py

```diff
@@ -1,29 +1,32 @@
+from typing import IO
+
 import hashlib
 import os
 
 import click
 
 
 @click.group()
-@click.pass_context
-def artifact(ctx):
+def artifact() -> None:
     pass
 
 
 @artifact.command()
 @click.option("--sha256")
 @click.pass_context
-def list(ctx, **kwargs):
+def list(ctx: click.Context, **kwargs: str) -> None:
     params = {k: v for k, v in kwargs.items() if v is not None}
     result = ctx.obj.call("artifacts_list", parameters=params)
     ctx.obj.output_result(result)
 
 
 @artifact.command()
 @click.option("--file", type=click.File("rb"), required=True)
-@click.option("--chunk-size", default=1000000, help="Chunk size in bytes (default is 1 MB)")
+@click.option(
+    "--chunk-size", default=1000000, type=int, help="Chunk size in bytes (default is 1 MB)"
+)
```

Comment:
Why the change here?

Reply:
It is adding `type=int` and i hope that will add input validation on the click side.
Not sure if the decorator will add type annotations to the variable passed to the command, but that would be super cool.